contrib/kiss-depends

```diff
@@ -1,9 +1,15 @@
 #!/bin/sh -ef
 # Display a package's dependencies
 
+# CRUX style usage using the current directory as the name of the package to be
+# operated on.
+[ "$1" ] || {
+    set -- "${PWD##*/}"
+}
+
 # Ignore shellcheck as we want the warning's behavior.
 # shellcheck disable=2015
-[ "$1" ] && kiss l "${1:-null}" >/dev/null || {
+kiss l "${1:-null}" >/dev/null || {
```

Comment:
i wonder if this should be `kiss list` since there is also `kiss link` and its just nice to be explicit?

Reply:
`kiss link` aka `kiss-link` is an external utility, `kiss l` aka. `kiss list` is built into the package manager itself. Not disagreeing with being explicit though.